utils/mil_tools/nodes/online_bagger.py

```diff
@@ -285,8 +297,8 @@ def set_bag_directory(self, filename=''):
         # If directory param is not set, default to $HOME/bags/<date>
         default_dir = self.dir
         if default_dir == None:
-            default_dir = os.path.join(os.environ['HOME'], 'bags' ,str(datetime.date.today()))
-
+            default_dir = os.path.join(os.environ['HOME'], 'bags')
+        default_dir = os.path.join(default_dir, str(datetime.date.today()))
```

Comment:
I added this as a hotfix at test day, but it should probably be more customizable. I suggest adding a boolean param "dated_folder" which, if true, will have this behavior of appending the data to the bag dir.